downloads/nightlies.md

```diff
@@ -48,4 +51,40 @@ Most users are advised to use the latest official release version of Julia.
   </tr>
   </tbody>
 </table>
+
+<script>
+document.addEventListener("DOMContentLoaded", function () {
+  // Get the table with class "downloads"
+  var downloadsTable = document.querySelector(".downloads");
+
+  // Check if the table is found
+  if (downloadsTable) {
+    // Get all the links in the table
+    var links = downloadsTable.querySelectorAll("a");
```

Comment:
Out of curiosity, how many elements does `links` contain?

Reply:
9